homeassistant/components/fibaro/__init__.py

```diff
@@ -454,59 +454,59 @@ def __init__(self, fibaro_device: DeviceModel) -> None:
         if not fibaro_device.visible:
             self._attr_entity_registry_visible_default = False
 
-    async def async_added_to_hass(self):
+    async def async_added_to_hass(self) -> None:
         """Call when entity is added to hass."""
         self.controller.register(self.fibaro_device.fibaro_id, self._update_callback)
 
-    def _update_callback(self):
+    def _update_callback(self) -> None:
         """Update the state."""
         self.schedule_update_ha_state(True)
 
     @property
-    def level(self):
+    def level(self) -> int | None:
         """Get the level of Fibaro device."""
         if self.fibaro_device.value.has_value:
             return self.fibaro_device.value.int_value()
         return None
 
     @property
-    def level2(self):
+    def level2(self) -> int | None:
         """Get the tilt level of Fibaro device."""
         if self.fibaro_device.value_2.has_value:
             return self.fibaro_device.value_2.int_value()
         return None
 
-    def dont_know_message(self, action):
+    def dont_know_message(self) -> None:
         """Make a warning in case we don't know how to perform an action."""
         _LOGGER.warning(
             "Not sure how to setValue: %s (available actions: %s)",
             str(self.ha_id),
             str(self.fibaro_device.actions),
         )
```

Comment:
Is this change correct, or is the warning print incorrect because it doesn't include the unknown action?

Reply:
Just thought I should not do different things in one PR. But you're absolutely right, it makes more sense to have the action within the log message. So I changed it accordingly.